src/ldc/eh/win32.d

```diff
@@ -113,103 +129,140 @@ extern(C) void _d_throw_exception(Object e)
 
 ///////////////////////////////////////////////////////////////
 
+__gshared CatchableType* ctCxxDThrowable;
+__gshared CatchableType* ctCxxDException;
+__gshared CatchableType* ctCxxDError;

```

Comment:
Good spot, removed. I wonder if it would be better to just use the C++ mangling for D aswell to avoid the duplication, but the exceptions will look like C++ in the debugger even more (i.e. showing "::" instead ".").